docs/entities/OfferReview.md

```diff
@@ -0,0 +1,24 @@
+# OfferReview
+
+## Descripción
+Representación de una resenya en la plataforma.
+
+## Propiedades
+* id: Identificador único de la reseña.
+* buyerId: Identificador único del comprador.
+* userImageUrl: URL de la imagen del usuario que escribió la reseña.
```

Comment:
There's no need to include the avatar of the reviewer in here. That information belongs to the user profile of the reviewer, which can be accessed from the relation via buyerId.